Frontend/components/services/events/hero.jsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from "@/components/ui/button";
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+
+const Hero = () => {
+  return (
+    <div className="w-screen 2xl:w-full -ml-4 2xl:ml-auto h-screen sm:h-[75vh] relative flex flex-col items-center justify-center gap-6 sm:gap-8 mt-[80px]">
+      <Image
+        src="/images/services/events/1.jpeg"
+        alt="hero"
+        width={1920}
+        height={1080}
+        className="object-cover w-full h-full"
+      />
+      <div className="absolute top-0 left-0 w-full h-full bg-secondary/50 flex flex-col items-center justify-center gap-4 px-4 md:px-8 ">
+        <h1 className="text-4xl sm:text-5xl md:text-7xl text-center text-background z-10 font-black column-text">
+          Unleash your inner coding genius at local developer gatherings!
```

Comment:
I have udjusted the font size and centered the text to avoid it covering the entire screen on large screens. @RuvimboRue